experiments/push_snapshot_to_s3.py

```diff
@@ -0,0 +1,71 @@
+import boto3
+import os
+import sys
+import logging
+from datetime import datetime
+
+logging.basicConfig(level=logging.INFO)
+
+
+logger = logging.getLogger(__name__)
+
+
+def upload_file_to_s3(file_path: str, bucket_name: str) -> None:
+    """
+    Uploads a file to an s3 bucket.
+    :param file_path: The path to the file to upload.
+    :param bucket_name: The name of the bucket to upload to.
+    """
+    s3_client = boto3.client("s3")
+    try:
+        s3_client.upload_file(file_path, bucket_name, os.path.basename(file_path))
```

Comment:
This requires already having a `~/.aws/credentials` file since boto3 looks for that.